toptal/picasso
# feat(Icon): [SPC-1436] add new arrow subdirectory icon

[SPC-1436]

### Description

Add the new BASE icon to Picasso, you can check it here: https://share.goabstract.com/4d95071b-964a-45f3-b13a-eae5a066ff29?mode=design&sha=9642220c68caba6cd449f1c4c29d58baf42c8fc5

### How to test

- Open Storybook, navigate to **Icons** section and check both new icons: 
  - ArrowSubdirectory16
  - ArrowSubdirectory24 


<img width="344" alt="Screen Shot 2021-11-18 at 15 28 37" src="https://user-images.githubusercontent.com/6453211/142475803-b1f24c76-df54-4fe7-9cb5-ea534c94fd92.png">


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[SPC-1436]: https://toptal-core.atlassian.net/browse/SPC-1436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
